cur/programming/lists/hof/ttt/05-winning-square.html

```diff
@@ -8,7 +8,7 @@
 
 <body>
 <h3> Can the Computer Win on this Move? </h3>
-<p>Converting a <em>position</em> into a <em>list of triples </em>will pay off as we start writing strategy rules. The overall structure of <code>ttt</code> will be (don't try to build this in Snap<em>!</em> yet; just read it)</p>
+<p>Let's call the current board our <em>position</em>; it's the current position of the game. Converting a <em>position</em> into a <em>list of triples </em>will pay off as we start writing strategy rules. The overall structure of <code>ttt</code> will be (don't try to build this in Snap<em>!</em> yet; just read it)</p>
```

Comment:
I felt that way too but most of the ttt images have "position" as the parameter where a board goes......so we could change it but I think we were going for consistency here.